tests/end_to_end_tests/pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+log_cli = True
+log_level = INFO
```

Comment:
I assume this can be overridden on per pytest run invocation basis in case it's needed?

Reply:
Yes, CLI arguments such as `--log-format`, `--log-level` has to override this config options. I haven't checked, but I believe that those ini files also have to respect the scope where they are located, so this particular file shouldn't have any effect outside of the tests/end-to-end-tests folder.